src/content/pages/sponsors.html

```diff
@@ -9,6 +9,7 @@
 {%- from 'components/donation.macro.html' import donationItem %}
 {%- from 'components/hero.macro.html' import hero %}
 
+{% set title = site.sponsors_page.title %}
```

Comment:
This shouldn’t be necessary? The page title is pulled using the hook value in base.html.

Reply:
Disclaimer: my knowledge of Eleventy is fairly limited.

If I remove this line, `title` is undefined.

I see `page_title` being pulled based on `hook` in `base.html`:

https://github.com/eslint/eslint.org/blob/6a26fa2c26d7f332ef2e564da6d753818bda5dca/src/_includes/layouts/base.html#L14

If I remove this line and change `hero({ title: title })` to `hero({ title: page_title })`, `page_title` is also undefined. Should `page_title` be in scope here from `base.html`?

Reply:
Ah I see. In this case, the call to hero should be passing `site.sponsors_page.title` instead of just `title`.

Reply:
Gotcha! I updated to do that in ab9854d. I also rebased on `main` to make sure this has the latest changes.